android/app/src/main/java/org/fossasia/openevent/adapters/DayScheduleAdapter.java

```diff
@@ -99,6 +105,45 @@ public void onBindViewHolder(DayScheduleViewHolder holder, int position) {
             holder.slotTrack.getBackground().setColorFilter(storedColor, PorterDuff.Mode.SRC_ATOP);
             holder.slotTrack.setText(sessionTrack.getName());
 
+            if(currentSession.isBookmarked()) {
```

Comment:
I think the same functionality of this block is present elsewhere, please share the code between two.

It becomes unmaintainable this way